packages/nbmodel/src/ymodel.ts

```diff
@@ -337,6 +359,44 @@ export class YMarkdownCell
   }
 }
 
-export const createSharedNotebook = (): nbmodel.ISharedNotebook => {
-  return new YNotebook();
-};
+/**
+ * Namespace for the factory methods.
+ */
+export namespace YNotebook {
+  export const createSharedNotebook = (): nbmodel.ISharedNotebook => {
```

Comment:
Maybe instead call it simply `create()`? Similarly to `cell.create()`.

Reply:
Yes, done.